### PR TITLE
Add notices.txt and update copyright line

### PR DESCRIPTION
**Pull request by stoskov**

### CODE REVIEW

File: notices.txt [hunk 1]
```diff
@@ -0,0 +1,89 @@
+=========================================================================
+==  NOTICE file corresponding to section 4 d of            	 	  ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for NativeScript Doctor v0.9					  ==
+=========================================================================
+
+NativeScript Doctor v0.9 (the �Product�)
```
Comment (rosen-vladimirov) [inline notices.txt]: https://github.com/NativeScript/nativescript-doctor/blob/master/package.json#L3 the current version is 0.12.0

File: notices.txt [hunk 2]
```diff
@@ -0,0 +1,89 @@
+=========================================================================
+==  NOTICE file corresponding to section 4 d of            	 	  ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for NativeScript Doctor v0.9					  ==
+=========================================================================
+
+NativeScript Doctor v0.9 (the �Product�)
+
+Copyright � 2016-2018 Telerik AD. All Rights Reserved.
```
Comment (rosen-vladimirov) [inline notices.txt]: what are these symbols: `Copyright � `

File: notices.txt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+==  NOTICE file corresponding to section 4 d of            	 	  ==
+==  the Apache License, Version 2.0,                                   ==
+==  in this case for NativeScript Doctor v0.9					  ==
+=========================================================================
+
+NativeScript Doctor v0.9 (the �Product�)
+
+Copyright � 2016-2018 Telerik AD. All Rights Reserved.
+
+For license information see the LICENSE.md file which accompanies this NOTICE.txt file.
```
Comment (rosen-vladimirov) [inline notices.txt]: the file is called `notices.txt`, but here it is mentioned as `NOTICE.txt`, is this a problem?

Comment (stoskov) [reply notices.txt]: This is a question I also had. I will rename it to notice.txt

File: notices.txt (hunk 2 above)
Comment (stoskov) [reply notices.txt]: Shawna's symbols :)

File: notices.txt (hunk 1 above)
Comment (stoskov) [reply notices.txt]: The problem is we have not had this file before. I had to create one for Sidekick. But now we have the Sidekick's notice.txt and you are right I that we could request new one.

**Pull request merged**